.gitignore

```diff
@@ -1,5 +1,4 @@
 dist
 package
-.slack/apps.json
-triggers/.gitignore
+.slack/
```

Comment:
When users create a new app from this template, they will get this `.gitignore` file. I'm wondering if our default stance should be to ignore `.slack/apps.json` and have users manually unignore it if they want to do collaboration, or if our default should be to include `.slack/apps.json`, and have folks add it to `.gitignore` when they are open sourcing / templatizing their app?

We may rethink collaboration before GA so apps.json could go moot by then.

Reply:
Yep, works for me. We still need to figure out the `app.json` and `apps-dev.json` as well. As we figure that out, we can revisit the versioning of `.slack/`